tests/src/rules/prefer-default-export.js

```diff
@@ -7,153 +7,382 @@ import { version as tsEslintVersion } from 'typescript-eslint-parser/package.jso
 const ruleTester = new RuleTester();
 const rule = require('../../../src/rules/prefer-default-export');
 
-ruleTester.run('prefer-default-export', rule, {
-  valid: [].concat(
-    test({
-      code: `
+const SINGLE_EXPORT_ERROR_MESSAGE = 'Prefer default export on a file with single export.';
+const ANY_EXPORT_ERROR_MESSAGE = 'Prefer default export to be present on every file that has export.';
+
+// rules with only single export
+const valid = [
+  test({
+    code: `
         export const foo = 'foo';
         export const bar = 'bar';`,
-    }),
-    test({
-      code: `
-        export default function bar() {};`,
-    }),
-    test({
```

Comment:
removing test cases is a red flag for a breaking change - can we keep all the existing ones with implicit defaults, and add *new* mostly-duplicate ones with the explicit target option?

Reply:
That was my intention, but I accidentally moved this specific test case without having its copy. I refactored the test cases to have existing ones (with implicit) untouched, and also added new duplicate ones with explicit target option set to 'any'.

Reply:
I brought back all inline test cases as it was before, so every test case which was present before is present now, too.